tejas-k3/tejas-kothari-portfolio
# ISSUE-8 Fix Project scripts configuration

### Description
- Fix Project scripts configuration in `package.json` to enable local development without any platform-specific modifications.

- Installed `cross-env` as a dev dependency.
  - It is a cross-platform solution that allows to set environment variables in a cross-platform manner.

- Added a script that works on both **Mac** and **Windows** platforms:
> "scripts": {
>     "start": "cross-env NODE_OPTIONS=--openssl-legacy-provider react-scripts start",
>     "build": "cross-env NODE_OPTIONS=--openssl-legacy-provider react-scripts build"
         ...
>   }

### Testing Instructions
- Run `npm start` or `npm run build` and verify if the command runs without any platform-specific modifications.




*** 
### NOTE:
- The above solution was not necessary in this case. however, it may prove useful in other Reactjs projects being developed in a cross-platform model.
- In our case I believe my system (Mac) was configured to use an old version of `node` and `npm` and so the general scripts were not working.
